src/controllers/IndexController.php

```diff
@@ -337,22 +337,42 @@ class="fb-main page--team-registration full-height fb-scroll">
                 <a href="#" id="custom-emblem-link">or upload your own</a>
               </h6>
               <div class="custom-emblem">
-                <input autocomplete="off" name="custom-emblem" id="custom-emblem-input" type="file" accept="image/*"/>
-                <img id="custom-emblem-preview" src=""></img>
+                <input
+                  autocomplete="off"
+                  name="custom-emblem"
+                  id="custom-emblem-input"
+                  type="file"
+                  accept="image/*"
+                />
+                <img
+                  id="custom-emblem-preview"
+                  src=""
+                  height={62}
+                  width={80}>
+                </img>
               </div>
               <div class="emblem-carousel">
                 <div id="custom-emblem-carousel-notice">
                   <div class="center-wrapper">
                     <h6>
-                      <a href="#" id="custom-emblem-clear-link">Clear your custom emblem to use a default emblem</a>
+                      <a href="#" id="custom-emblem-clear-link">
+                        Clear your custom emblem to use a default emblem
```

Comment:
Good catch!  Will go ahead and add the translation wrapper for the various strings that were missed.